src/core/ext/transport/binder/server/binder_server_credentials.cc

```diff
@@ -12,27 +11,25 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//
 
-#include <grpc/grpc.h>
-#include <grpc/support/log.h>
-#include <gtest/gtest.h>
-
-#include "test/core/util/test_config.h"
-
-namespace grpc_core {
-namespace testing {
-namespace {
-TEST(SmokeTest, Empty) { gpr_log(GPR_INFO, __func__); }
-}  // namespace
-}  // namespace testing
-}  // namespace grpc_core
-
-int main(int argc, char** argv) {
-  ::testing::InitGoogleTest(&argc, argv);
-  grpc::testing::TestEnvironment env(argc, argv);
-  grpc_init();
-  int ret = RUN_ALL_TESTS();
-  grpc_shutdown();
-  return ret;
+#include <grpc/impl/codegen/port_platform.h>
+
+#include "src/core/ext/transport/binder/server/binder_server_credentials.h"
+
+#include <grpcpp/security/server_credentials.h>
+
+#ifdef GPR_ANDROID
+
+namespace grpc {
+namespace experimental {
+
+std::shared_ptr<ServerCredentials> BinderServerCredentials() {
+  return std::shared_ptr<ServerCredentials>(
+      new grpc::internal::BinderServerCredentialsImpl<
+          grpc_binder::TransactionReceiverAndroid>());
```

Comment:
Need to include corresponding header, otherwise

src/core/ext/transport/binder/server/binder_server_credentials.cc:29:24: error: no member named 'TransactionReceiverAndroid' in namespace 'grpc_binder'

Reply:
Thanks! I forgot to try building in Android.